chef/cookbooks/network/recipes/default.rb

```diff
@@ -16,6 +16,8 @@
 
 return if node[:platform_family] == "windows"
 
+dirty = false
```

Comment:
In any case, we dont do a node.save in there so it wont be needed. The point of the PR is to avoid doing node.save, not to avoid saving attributes. When we do a node.save, supposedly its because we need that value to be in the node instantly, so other nodes may access it (or searches), so in this case there doesnt seem to be a need of making that value available anywhere else, so no node.save in here :)